### PR TITLE
Update language tags

### PR DESCRIPTION
**Pull request by Rabenherz112**
I have noticed that some of the software I use does not include all the languages in their installation options (especially Docker or Kubernete). This can be quite frustrating at times because I may install a particular software only to later discover that there is a better alternative available, but I didn't consider it initially because it was not compatible with my environment.

Therefore, the purpose of this pull request is to enhance the language support of the software by adding the languages that are currently supported along with their respective installation instructions. While I may still miss some programs or languages, this effort should significantly improve the situation.

Please note that this pr is currently a draft, and I will mark it as ready once I have completed the list. However, I wanted to provide an update on the progress and be available for any discussions or address any concerns that may arise.

The fact that the table of contents was edited is a bug in VS Code (it does this for some reason with every commit), and once I'm done, the whole thing will be reverted. (https://github.com/awesome-selfhosted/awesome-selfhosted/commit/ad658c943890b19230012653fcac8db540bf9986)

Edit:
Note for me, next up Communication - IRC

### CODE REVIEW

**Comment by nodiscc** [comment]
Thanks for taking the time to do this. I suggest doing it in several not-too-large batches, to minimize the risk of hard to resolve merge conflicts that will arise if other PRs on the same sections are merged while you work on this.

File: README.md [hunk 1]
```diff
@@ -121,17 +123,17 @@ See [Contributing](.github/CONTRIBUTING.md).
 
 _Related: [Database Management](#database-management), [Personal Dashboards](#personal-dashboards)_
 
-- [Ackee](https://ackee.electerious.com) - Self-hosted analytics tool for those who care about privacy. ([Demo](http://demo.ackee.electerious.com), [Source Code](https://github.com/electerious/Ackee)) `MIT` `Nodejs`
+- [Ackee](https://ackee.electerious.com) - Self-hosted analytics tool for those who care about privacy. ([Demo](http://demo.ackee.electerious.com), [Source Code](https://github.com/electerious/Ackee)) `MIT` `Nodejs/Docker/K8S`
 - [Aptabase](https://aptabase.com/) - Open Source, Privacy-First and Simple Analytics for Mobile and Desktop apps. SDKs available for Swift, Kotlin, Flutter and many others! ([Source Code](https://github.com/aptabase/aptabase)) `AGPL-3.0` `Docker`
```
Comment (KieranRobson) [inline README.md:127]:
```suggestion
- [Aptabase](https://aptabase.com/) - Open source, privacy first and simple analytics for mobile and desktop apps. SDKs available for Swift, Kotlin, Flutter and many others! ([Source Code](https://github.com/aptabase/aptabase)) `AGPL-3.0` `Docker`
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
I dont want to do an entire PR for this but noticed it when looking through the list if you wanted to change it, feel free or I can do a PR. Thanks!

**Comment by Rabenherz112** [comment]
> I suggest doing it in several not-too-large batches/PRs, to minimize the risk of hard to resolve merge conflicts that will arise if other PRs on the same sections are merged while you work on this.

Sounds good, I'll make sure to divide the task into smaller batches Thanks for the heads up! 👍

**Comment by Rabenherz112** [comment]
Commit https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3945/commits/bb22831a678d335c08c3b09d95ee17fe040bc1f0 & https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3945/commits/b7914b9982e14729ac6e581478c3d3f5dbf387cb don't change the language tag of the project but instead one time the website link and one time the repo.
If I should exclude these two changes into an extra PR let me know.

**Review by nodiscc — APPROVED** [review]
Thanks a lot

**Pull request merged**